devtools/ferc1-etl-debug.ipynb

```diff
@@ -23,10 +23,23 @@
   },
```

Comment:
<p>Note that this simple loop is failing because the PlantsSteamFerc1TableTransformer has non-standard inputs -- it also needs the <code>transformed_fuel</code> table. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/2356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>